scopus-import/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -1754,19 +1754,9 @@ private Contributor findContributorByName(String givenName, List<Contributor> co
     }
 
     private void checkCollaborationName(CollaborationTp collaboration, List<Contributor> contributors) {
-        var optionalContributor = findContributorBySequence(collaboration.getSeq(), contributors);
-        if (Integer.parseInt(collaboration.getSeq()) < contributors.size()) {
-            var contributor = optionalContributor.getFirst();
+        var contributor = findContributorByName(collaboration.getIndexedName(), contributors);
 
-            assertEquals(1, optionalContributor.size());
-            assertEquals(collaboration.getIndexedName(), contributor.getIdentity().getName());
-        }
-    }
-
-    private List<Contributor> findContributorBySequence(String sequence, List<Contributor> contributors) {
-        return contributors.stream()
-                   .filter(contributor -> sequence.equals(String.valueOf(contributor.getSequence())))
-                   .collect(Collectors.toList());
+        assertEquals(collaboration.getIndexedName(), contributor.getIdentity().getName());
```

Comment:
If we already found the contributor by name it's a bit strange to assert that it has the same name. Change to assertThat(contributor, is(notNullValue())?